mitosheet/switch.py

```diff
@@ -392,9 +283,11 @@ def switch(new_package):
     if new_package == 'mitosheet2':
       open('package.json', 'w').write(MITOSHEET_TWO_PACKAGE_JSON)
     elif new_package == 'mitosheet':    
-      open('package.json', 'w').write(MITOSHEET_PACKAGE_JSON)
+      open('package.json', 'w').write(MITOSHEET_PACKAGE_JSON.replace('REPLACE_WITH_PACKAGE_NAME_WITH_REPLACE', 'mitosheet'))
```

Comment:
This text doesn't make sense to me. 

Should it say: "REPLACE_WITH_PACKAGE_NAME" instead of "REPLACE_WITH_PACKAGE_NAME_WITH_REPLACE"?